cmd2/cmd2.py

```diff
@@ -34,6 +34,7 @@
 import inspect
 import os
 import pickle
+import pydoc
```

Comment:
I've never heard of `pydoc` before.  Is it a built-in module?

Reply:
I looked it up and it appears to be a built-in.

Reply:
Coming from a background in C/C++, the sheer volume of stuff built into Python's standard library continues to amaze me even after working with it for over a decade!